dali/pipeline/operator/op_schema.h

```diff
@@ -47,6 +47,12 @@ struct DefaultedArgumentDef {
   Value *default_value;
 };
 
+struct DeprecatedArgumentDef {
+  std::string deprecated_str;
```

Comment:
How about just printing this in Python, too?

Reply:
renamed_to/ignore are not only for printing but also to know whether to automatically redirect the deprecated argument to the new name, or totally remove it from the arguments (of course while warning the user that he/she is using a deprecated argument)

Reply:
OK.